client/mksubvolume.cc

```diff
@@ -377,7 +377,14 @@ doit()
     catch (const runtime_error_with_errno& e)
     {
 	if (e.error_number == EEXIST && force)
+	{
+	    const string path = string(tmp_mountpoint.mountpoint.get()).append("/").append(subvolume_name);
+
+	    struct stat sb;
+	    if (stat(path.c_str(), &sb) == 0 && !is_subvolume(sb))
```

Comment:
Failure of stat is not handled.

Reply:
And it should be lstat anyway...

Reply:
Error condition reversed to only accept "good" case.